### DISCUSSION TITLE
PHP Deprecated:  Optional parameter $force declared before required parameter $testType is implicitly treat ed as a required parameter

### DISCUSSION THREAD

**Discussion by RealMrHex** [livewire/livewire#7318] [7. Bugs (archived)] [answered]
### Livewire version

3.2

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

PHP Deprecated:  Optional parameter $force declared before required parameter $testType is implicitly treat
ed as a required parameter in /var/www/html/vendor/livewire/livewire/src/Features/SupportConsoleCommands/Commands/MakeCommand.php on line 111

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment, marked as answer]
@RealMrHex there is already an open issue #7315 and I just submitted this PR #7319 to fix 🙂

**Reply by RealMrHex** [reply]
Gotcha, tnx
